pkg/cmd/cli/cmd/request_project.go

```diff
@@ -52,7 +52,7 @@ func NewCmdRequestProject(name, fullName, ocLoginName, ocProjectName string, f *
 	cmd := &cobra.Command{
 		Use:     fmt.Sprintf("%s NAME [--display-name=DISPLAYNAME] [--description=DESCRIPTION]", name),
 		Short:   "Request a new project",
-		Long:    fmt.Sprintf(requestProjectLong, ocLoginName, ocProjectName),
+		Long:    requestProjectLong,
```

Comment:
@deads2k Want to make sure this doesn't change the intent here. `requestProjectLong` doesn't have any formatting directives.